.travis.yml

```diff
@@ -15,5 +15,5 @@ before_install:
 install: raco pkg install --deps search-auto $TRAVIS_BUILD_DIR
 
 script:
- - raco test $TRAVIS_BUILD_DIR
+ - raco test --submodule integration-test $TRAVIS_BUILD_DIR
```

Comment:
You'll need *two* test commands, the original one that runs unit tests and this one to run integration tests. Otherwise any (module+ test body ...) forms elsewhere in the package won't run. I haven't actually written any tests though, so technically not a big deal.